fr.etnadji.dispremi.json

```diff
@@ -0,0 +1,65 @@
+{
+    "app-id" : "fr.etnadji.dispremi",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dispremi",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri",
+	"--filesystem=xdg-config",
```

Comment:
The change is supposed to be done from the app side btw, in the flatpak manifest you can complete drop this line. Which toolkit are you using to build the application? GTK?

Reply:
Yes, GTK + pygi. In the app, the configuration file path is not hardcoded as I use appdirs module to get the folder path.

Dropping this line : because of `--filesystem=host` I suppose